pymeasure/instruments/agilent/agilentL4534A.py

```diff
@@ -0,0 +1,333 @@
+            strict_discrete_set(value['coupling'], COUPLING_VALUES)
+            strict_discrete_set(value['filter'], FILTER_VALUES)
+            return value
+    
+        config = Instrument.control(
+            "CONF:CHAN:ATTR? (@{ch})",
+            "CONF:CHAN:ATTR (@{ch}),%s",
+            """Control Channel configuration with dict containing range (in V), coupling, and filter""",
+            set_process=lambda v: '{:.3g},{},{}'.format(
+                v['range'].m_as(ureg.V), v['coupling'], v['filter']),
+            get_process=_get_channel_config_process,
+            validator=_validate_channel_config
+        )
+
+        range = Instrument.control(
+            "CONF:CHAN:RANG? (@{ch})",
+            "CONF:CHAN:RANG (@{ch}),%s",
+            """Control Voltage range for this channel
+            Ranges: 0.25, 0.5, 1, 2, 4, 8, 16, 32, 128, 256
+            """,
+            set_process=lambda v: Decimal(v.m_as(ureg.V)).to_eng_string(
+            ),  # send the value as V to the device
+            get_process=lambda v: ureg.Quantity(v, ureg.V),  # convert to quantity
+            validator=strict_discrete_set,
+            values=VOLTAGE_RANGE_VALUES
+        )
+
+        coupling = Instrument.control(
+            "CONF:CHAN:COUP? (@{ch})",
+            "CONF:CHAN:COUP (@{ch}),%s",
+            """Comtrol channel coupling {AC|DC}""",
+            validator=strict_discrete_set,
+            values=COUPLING_VALUES
+        )
+
+        filter = Instrument.control(
+            "CONF:CHAN:FILT? (@{ch})",
+            "CONF:CHAN:FILT (@{ch}),%s",
+            """Control Filter for this channel (LP_20_MHZ, LP_2_MHZ, LP_200_KHZ)""",
+            validator=strict_discrete_set,
+            values=FILTER_VALUES
+        )
+
+        def _read_data_block(self):
+            # Data block has header of form '#<n><n digits>',
+            # where <n>specifies the number of digits used to indicate the size of the block,
+            # and the <n digits> indicate the block size in bytes.
+            digits = self.read_bytes(2).decode()
+            if not digits.startswith('#'):
+                raise Error()
+            bytes = int(self.read_bytes(int(digits[1])).decode())
+            # Read specified number of bytes + newline terminator
+            buf = memoryview(self.read_bytes(bytes+1))
+            # Return view that strips off newline to contain only desired bytes
+            return buf[:-1]
+
+        @property
+        def voltage(self) -> typing.NDArray[np.float32]:
+            """Get voltage measurements for this channel"""
+            self.write(f'FETC:WAV:VOLT? (@{self.id})')
+            data = self._read_data_block()
+            return ureg.Quantity(np.frombuffer(data, dtype='>f4'), ureg.V)
+
+        @property
+        def adc(self) -> typing.NDArray[np.int16]:
+            """Get raw ADC measurements for this channel in counts (-32,767,+32,767) in current voltage range"""
```

Comment:
The use of commas is misleading. You could drop them, or use a semicolon in the middle:
```suggestion
            """Get raw ADC measurements for this channel in counts (-32767, +32767) in current voltage range"""
```